lnrpc/rpc.proto

```diff
@@ -485,7 +485,9 @@ service Lightning {
     */
     rpc CloseChannel (CloseChannelRequest) returns (stream CloseStatusUpdate) {
         option (google.api.http) = {
-            delete: "/v1/channels/{channel_point.funding_txid_str}/{channel_point.output_index}"
+            delete:
```

Comment:
commit message: we use subsystems as prefixes, but lnd is not a subsystem

Reply:
Good to know, have removed lnd from commit message.